tests/atlocal.in

```diff
@@ -132,6 +132,14 @@ runroot()
     snapshot exec "$@"
 }
 
+runpython()
```

Comment:
Hmm, this adds yet another way to run python in the test-suite :sweat_smile: (since we have `RPMPY_TEST` already). Could that macro be made more generic so that it can run any python script, not just `test.py`?

Alternatively, maybe rename this wrapper to something more generic (since it doesn't seem to be super specific to running python)?

Reply:
Note that either is can be fixed later, I'm fine with this wrapper for now, it does the job.

Reply:
Yep, indeed. But let's not bother with it here, this can always be done later, it's just a silly little shell function anyway.

Reply:
Yep. Just noting for the record that this is far from ideal, we shouldn't be duplicating the ASAN magic around like this. It'll take some refactoring of the fakechroot-era python test-runner etc though.